packages/alfa-style/src/property/clip.ts

```diff
@@ -0,0 +1,26 @@
+import { Keyword, Rectangle } from "@siteimprove/alfa-css";
+import { Parser } from "@siteimprove/alfa-parser";
+
+import { Property } from "../property";
+
+const { either } = Parser;
+
+export namespace Clip {
+  export type Auto = Keyword<"auto">;
+
+  export type Specified = Auto | Rectangle;
+
+  export type Computed = Specified;
+}
+
+/**
+ * @see https://drafts.fxtf.org/css-masking/#clip-property
+ */
+export const Clip: Property<
+  Clip.Specified,
+  Clip.Computed
+> = Property.of(
+  Keyword.of("auto"),
+  either(Keyword.parse("auto"), Rectangle.parse),
+  (style) => style.specified("clip")
```

Comment:
Specs says computed is "as specified". Feels a bit weird to not absolutise anything. 🤷